collections/ansible_collections/netris/controller/plugins/modules/netris_login.py

```
@@ -0,0 +1,67 @@
+            validate_certs=dict(type="bool", default=True),
+        ),
+        supports_check_mode=False,
+    )
+    base = module.params["url"].rstrip("/")
+    auth_url = base + "/api/auth"
+    data = json.dumps({
+        "user": module.params["username"],
+        "password": module.params["password"],
+        "auth_scheme_id": 1,
+    }).encode("utf-8")
+    req = urllib.request.Request(auth_url, data=data, method="POST", headers={"Content-Type": "application/json"})
```

Ruff (0.15.7)

[error] Audit URL open for permitted schemes. Allowing use of `file:` or custom schemes is often unexpected.

(S310)